src/common/reduction/matching/GapMatcher.cpp

```diff
@@ -18,6 +18,14 @@ void GapMatcher::set_minimum_time_gap(uint64_t minimum_time_gap) {
   minimum_time_gap_ = minimum_time_gap;
 }
 
+void GapMatcher::set_split_multi_events(bool split_multi_events,
```

Comment:
this is not the ECDC coding standard, should be
setSplitMultiEvents

depending on the coding style in this file

Reply:
Addressing in restyle branch